changelog/unreleased/kong/http-log-concurrent-optimize.yml

```diff
@@ -0,0 +1,4 @@
+message: |
+  **HTTP-Log**: When `queue.max_batch_size` is 1, log entries are now sent in separate, parallel HTTP requests.  Previously, they were sent sequentially in FIFO order.
```

Comment:
```suggestion
  **HTTP-Log**: When `queue.max_batch_size` is explicitly set to `1`, log entries are now sent via concurrent HTTP requests. Previously, they were sent sequentially in FIFO order.
```

Reply:
The default value of max_batch_size is 1. So words `explicitly set` is not applicable at this case.